hazelcast/src/main/java/com/hazelcast/internal/nearcache/impl/invalidation/RepairingTask.java

```diff
@@ -58,55 +55,54 @@
+    // only used for testing
+    static final HazelcastProperty MIN_RECONCILIATION_INTERVAL_SECONDS
+            = new HazelcastProperty("hazelcast.invalidation.min.reconciliation.interval.seconds", 1, SECONDS);
 
-    static final long MIN_RECONCILIATION_INTERVAL_SECONDS = 30;
-    static final long GET_UUID_TASK_SCHEDULE_MILLIS = 500;
+    static final long RESCHEDULE_FAILED_INITIALIZATION_AFTER_MILLIS = 500;
     static final long HALF_MINUTE_MILLIS = SECONDS.toMillis(30);
 
-    final long reconciliationIntervalNanos;
     final int maxToleratedMissCount;
+    final long reconciliationIntervalNanos;
 
-    private final ConcurrentMap<String, RepairingHandler> handlers = new ConcurrentHashMap<String, RepairingHandler>();
-    private final AtomicBoolean running = new AtomicBoolean(false);
-
-    private final MetaDataFetcher metaDataFetcher;
-    private final TaskScheduler scheduler;
-    private final SerializationService serializationService;
-    private final MinimalPartitionService partitionService;
     private final int partitionCount;
-    private final AtomicReferenceArray<UUID> partitionUuids;
     private final String localUuid;
     private final ILogger logger;
+    private final TaskScheduler scheduler;
+    private final MetaDataFetcher metaDataFetcher;
+    private final SerializationService serializationService;
+    private final MinimalPartitionService partitionService;
+    private final AtomicBoolean running = new AtomicBoolean(false);
+    private final ConcurrentMap<String, RepairingHandler> handlers = new ConcurrentHashMap<String, RepairingHandler>();
 
     private volatile long lastAntiEntropyRunNanos;
 
     public RepairingTask(HazelcastProperties properties, MetaDataFetcher metaDataFetcher, TaskScheduler scheduler,
                          SerializationService serializationService, MinimalPartitionService partitionService, String localUuid,
                          ILogger logger) {
-        this.reconciliationIntervalNanos = SECONDS.toNanos(checkAndGetReconciliationIntervalSeconds(properties));
-        this.maxToleratedMissCount = checkMaxToleratedMissCount(properties);
-
+        this.reconciliationIntervalNanos = SECONDS.toNanos(checkedReconciliationIntervalSeconds(properties));
+        this.maxToleratedMissCount = checkedMaxToleratedMissCount(properties);
         this.metaDataFetcher = metaDataFetcher;
         this.scheduler = scheduler;
         this.serializationService = serializationService;
         this.partitionService = partitionService;
         this.partitionCount = partitionService.getPartitionCount();
-        this.partitionUuids = new AtomicReferenceArray<UUID>(partitionCount);
         this.localUuid = localUuid;
         this.logger = logger;
     }
 
-    private int checkMaxToleratedMissCount(HazelcastProperties properties) {
+    private int checkedMaxToleratedMissCount(HazelcastProperties properties) {
```

Comment:
I think `getMaxToleratedMissCount()` is sufficient here, the checks are implementation details, aren't they?